source/_integrations/homematic.markdown

```diff
@@ -223,7 +223,7 @@ The values of variables are polled from the CCU/Homegear in an interval of 30 se
 
 ### Events
 
-When HomeMatic devices change their state or some other internal value, the CCU/Homegear sends event messages to Home Assistant. These events are automatically parsed and the entities in Home Assistant are updated. However, you can also manually use these events to trigger automations. Two event-types are available:
+When Homematic devices change their state or some other internal value, the CCU/Homegear sends event messages to Home Assistant. These events are automatically parsed and the entities in Home Assistant are updated. However, you can also manually use these events to trigger automations. Two event-types are available:
```

Comment:
I see that the spelling is used mixed: https://www.eq-3.de/produkte/homematic.html